pkg/loadbalancer/maps/test_utils.go

```diff
@@ -112,10 +112,12 @@ func DumpLBMaps(lbmaps LBMaps, sanitizeIDs bool, customizeAddr func(net.IP, uint
 
 		switch v := revValue.(type) {
 		case *RevNat4Value:
-			addr = customizeAddr(v.Address.IP(), v.Port)
+			addrCluster, _ := types.AddrClusterFromIP(v.Address.IP())
```

Comment:
Maybe we could use the panic variant of this call since this in test (and same for other variant of this in the rest of the PR)? It doesn't seems to be quite probable that this happen but at least it would hard fail just in case it does somehow :man_shrugging: 
```suggestion
			addrCluster := types.MustAddrClusterFromIP(v.Address.IP())
```

Reply:
Yeah, makes sense. I thought there could be cases in which the address was nil, which would have caused that to panic, but that doesn't seem to be happening, so I've just switched it.